sunpy/map/sources/sdo.py

```diff
@@ -121,4 +123,31 @@ def observatory(self):
     @classmethod
     def is_datasource_for(cls, data, header, **kwargs):
         """Determines if header corresponds to an HMI image"""
-        return str(header.get('instrume', '')).startswith('HMI')
+        return (str(header.get('TELESCOP', '')).endswith('HMI') and
+                not HMISynopticMap.is_datasource_for(data, header))
+
+
+class HMISynopticMap(HMIMap):
+
+    def __init__(self, data, header, **kwargs):
+        super().__init__(data, header, **kwargs)
+
+        if self.meta['cunit1'] == 'Degree':
+            self.meta['cunit1'] = 'degree'
+
+        if self.meta['cunit2'] == 'Sine Latitude':
+            self.meta['cunit2'] = 'degree'
+
+            # Instead of the spacing in sin(lat), this should be 180/pi times
+            # that value (see Thompson 2005)
```

Comment:
I recommend either (or both) specifically pointing to Section 5.5 in that paper or stating explicitly that this map uses the cylindrical equal-area (CEA) projection.